frontend/packages/knative-plugin/src/utils/create-channel-utils.ts

```diff
@@ -66,8 +66,7 @@ export const useChannelList = (namespace: string): ChannelListProps => {
       })
       // eslint-disable-next-line no-console
       .catch((err) => console.warn(err.message));
```

Comment:
Can you please add a console warning msg here as well

Reply:
Added